.github/workflows/greeting.yaml

```diff
@@ -0,0 +1,27 @@
+name: Greeting
+on:
+  pull_request_target:
+    types:
+      - opened
+
+jobs:
+  greeting:
+    permissions:
+      pull-requests: write
+    name: Greeting comment upon PR from fork
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name != github.repository
+
+    steps:
+      - name: Issue the greeting comment
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
is there an org token we can replace this generic token with?

Reply:
i could use the same one that gets used to bump the sysdig-deploy version, but it's called `DEPLOY_VERSION_CHANGE_TOKEN` which is not what we are doing here

or request a new one?  seems overkill tbh